modules/ROOT/pages/faq.adoc

```diff
@@ -377,3 +377,53 @@ storage:
         inline: |
           PubkeyAcceptedKeyTypes=+ssh-rsa
 ----
+
+== Why do I get SELinux denials after updates if I have local policy modifications?
+
+Currently the OSTree and SELinux tooling conflict a bit. If you have
+permanently applied local policy modifications then policy updates
+delivered by the OS will no longer apply; your policy stays frozen.
+This means any policy "fixes" needed to enable new functionality will
+not get applied. See
```

Comment:
Should we show the `ostree admin config-diff | grep selinux` check here so they can easily see if they're affected? It's in the tracker issue, but it's kinda nested in there.

Reply:
yeah that sounds like a good idea